lib/lwm2mUtils.js

```diff
@@ -32,6 +32,7 @@ function parseObjectUriList(payload) {
     return payload
```

Comment:
Please, add an entry to CHANGES_NEXT_RELEASE corresponding to the change introduced in this PR.

Reply:
fixed by 800a6d23a9025d02156474ddce6cc51f0de448bd